server.py

```diff
@@ -0,0 +1,149 @@
+import json
+from six.moves.urllib.request import urlopen
+from functools import wraps
+
+from flask import Flask, request, jsonify, _request_ctx_stack
+from flask_cors import cross_origin
+from jose import jwt
+
+app = Flask(__name__)
+
+with open('config.json') as json_data_file:
+    data = json.load(json_data_file)
+
+API_AUDIENCE = data["API_AUDIENCE"]
+TENANT_ID = data["TENANT_ID"]
+
+# Error handler
+class AuthError(Exception):
+    def __init__(self, error, status_code):
+        self.error = error
+        self.status_code = status_code
+
+@app.errorhandler(AuthError)
+def handle_auth_error(ex):
+    print('handling error')
+    response = jsonify(ex.error)
+    response.status_code = ex.status_code
+    return response
+
+# Format error response and append status code
+
+
+def get_token_auth_header():
```

Comment:
In general this can be more powerful if it can be isolated into a library. Need not be pip installable if its too simple, but if it is a library and since it is Azure AD these decoraters can be used across many Python SDK's.